pkg/collector/corechecks/net/network/network.go

```diff
@@ -313,8 +731,12 @@ func Factory() option.Option[func() check.Check] {
 }
 
 func newCheck() check.Check {
+	procfsPath := "/proc"
+	if pkgconfigsetup.Datadog().IsConfigured("procfs_path") {
+		procfsPath = pkgconfigsetup.Datadog().GetString("procfs_path")
```

Comment:
Let's use the config component instead of accessing the global config object via `pkgconfigsetup.Datadog()`. You can see here, that we have access to the config component. 

https://github.com/DataDog/datadog-agent/blob/main/pkg/commonchecks/corechecks.go#L60

https://github.com/DataDog/datadog-agent/blob/main/pkg/commonchecks/corechecks.go#L89

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/33190/commits/006c7bfe459c569d31d362ff93e72d205eb09a34